build/matches-in-text.js

```diff
@@ -18,6 +18,14 @@ function* findMatchesInText(needle, haystack, { attribute = null } = {}) {
   }
 }
 
+function findFirstMatchInText(needle, haystack) {
+  const index = haystack.indexOf(needle);
+  const left = haystack.substring(0, index);
+  const line = left.split("\n").length;
+  const column = left.length - left.lastIndexOf("\n");
+  return { line, column };
+}
```

Comment:
Perhaps this implementation is better. I'm not sure. If it is, we can discuss that. But if we're going to have to keep this, I think it should be a wrapper on `findMatchesInText` instead of a similar'ish re-implementation.

Reply:
I ran into crashes in `haystack.matchAll(rex)` for translated content. It's also much cheaper and faster for that use case. If you have strong feelings here, I can reproduce it and attach the trace.

Reply:
But it's a big change. If something existing is broken, we should fix it rather than adding a new alternative to the mix. 
I can take a look if you think it'll help.

Reply:
I'm not sure how to address this. I have a build running with this reverted. I it works I'll just revert it. In general it was confusing how it's used in https://github.com/mdn/yari/pull/3008/files/80eda6e710c7fb63280f6a071b501d6497e25139#diff-324a90fbbc2c406cf9b57a13045cc35ffa34102f7ae9a3eb4ba642b6fbb98780L339

We only want to find a single match but take the _last_ one (which probably is the only one).

Reply:
It's weird that it picks the last one. I need to refresh my memory about if that was intentional or a bug. 

> I it works I'll just revert it.

Is that a typo of "IF it works [I'll revert]" or "IT works [I'll revert]"?

Reply:
https://github.com/mdn/yari/runs/1972103598?check_suite_focus=true looks like it worked.

Restored